### ISSUE TITLE
Deep understanding for Riemannian Opt (Gradient Descent)

### ISSUE THREAD

**Issue by mbaddar1** [mbaddar1/genmodel#24]
Optimization Techniques on
Riemannian Manifolds
Steven T. Smith
https://arxiv.org/pdf/1407.5965.pdf 

AUTOMATIC DIFFERENTIATION FOR RIEMANNIAN OPTIMIZATION ON
LOW-RANK MATRIX AND TENSOR-TRAIN MANIFOLDS
https://arxiv.org/pdf/2103.14974.pdf

Tensor Train Decomposition on TensorFlow (T3F)
https://arxiv.org/pdf/1801.01928.pdf
